### ISSUE TITLE
Can't import v2

### ISSUE THREAD

**Issue by guregu** [biscuit-auth/biscuit-go#104]
I think v2 might be released improperly.

```bash
$ go get github.com/biscuit-auth/biscuit-go@v2.0.0
go: downloading github.com/biscuit-auth/biscuit-go/v2 v2.0.0
go: go.mod has non-.../v2 module path "github.com/biscuit-auth/biscuit-go" (and .../v2/go.mod does not exist) at revision v2.0.0

$ go get github.com/biscuit-auth/biscuit-go/v2@v2.0.0
go: github.com/biscuit-auth/biscuit-go@v2.0.0: invalid version: module contains a go.mod file, so module path must match major version ("github.com/biscuit-auth/biscuit-go/v2")
```

AFAIK these are the two options:
- Add a folder called `v2` to the project root, copy `go.mod` there and tack on `/v2` to the path
- Add a branch called `v2`, add `/v2` to the module path in `go.mod`

TL;DR: this needs `/v2` at the end: https://github.com/biscuit-auth/biscuit-go/blob/main/go.mod#L1, and I think it either needs to be in a `v2` branch or a `v2` folder (not sure if v2 from main branch works). The [internal imports for the datalog package, etc.](https://github.com/biscuit-auth/biscuit-go/blob/85592e25c7512d77897fd07d6a2d40f66ff393b9/biscuit.go#L14-L15), need /v2 in their imports as well.

**Comment by hagmonk** [comment]
I think this is still busted until someone tags the right version per https://github.com/biscuit-auth/biscuit-go/pull/106#issuecomment-1347379877

**Comment by Pranay-Munshi** [comment]
Is this issue fixed or not?
I tried to do go get github.com/biscuit-auth/biscuit-go/v2@370cd7c which was successful but in imports I am not able to use above path.
I tried below - 
"github.com/biscuit-auth/biscuit-go/v2@370cd7c" but it is saying invalid import path.

**Comment by Geal** [comment]
Right, thanks for the report, looking into it

**Comment by Geal** [comment]
I pushed a new tag, let me know if that fixes it for you

**Comment by chilarai** [comment]
Thanks. I did not face any issue this time while using the following
`go get github.com/biscuit-auth/biscuit-go/v2`

**Comment by guregu** [comment]
Works great now, thank you!
